src/renderers/shaders/ShaderChunk/global_fragment.glsl

```diff
@@ -0,0 +1 @@
+// template for injecting varying, uniform and functions above main below other includes
```

Comment:
Just that we're on the same page, if i understand how the hooks work, some will have the same names as chunks, but some will be new? I think `VERTEX_UNIFORM` you suggested may be exactly this.